packages/@aws-cdk/aws-elasticloadbalancing/README.md

```diff
@@ -18,7 +18,9 @@ classic load balancers.
 
 Load balancers send traffic to one or more AutoScalingGroups. Create a load
 balancer, set up listeners and a health check, and supply the fleet(s) you want
-to load balance to in the `targets` property.
+to load balance to in the `targets` property. We also need to set `internetFacing` as `true` on
```

Comment:
Can this be phrased without `we` here? this doesn't quite match the tone of the readme.